### DISCUSSION TITLE
Dart/Flutter Rewrite

### DISCUSSION THREAD

**Discussion by trymeouteh** [damus-io/damus#1916] [General]
This is a great app and I am only suggesting rewriting the app with dart/flutter to allow the app to be compilled and released for mobile and desktop.

Flutter allows you to create apps for Windows, MacOS, Linux, Android and iOS. More devices, more users using nostr and all done with one codebase to manage.

**Comment by jb55** [comment]
we already have a multiplatform re-write in the works in rust:

https://github.com/damus-io/notedeck
